tiledb/sm/misc/types.h

```diff
@@ -109,6 +117,9 @@ class Range {
 /** An N-dimensional range, consisting of a vector of 1D ranges. */
 typedef std::vector<Range> NDRange;
 
+/** A value as a vector of bytes. */
+typedef std::vector<uint8_t> Value;
```

Comment:
Could we rename this to something less generic? Maybe something like `ByteVecValue`?

Reply:
Sure thing, I can do that change.